repos.yaml

```diff
@@ -230,8 +230,17 @@ repositories:
     maintainers:
       - email: rafal@hazelcast.com
         name: Rafal Leszko
-      - email: guglielmo@hazelcast.com
-        name: Guglielmo Nigri
+      - email: hasan@hazelcast.com
+        name: Hasan Celik
+      - email: mesut@hazelcast.com
+        name: Mesut Celik
+  - name: hazelcast-jet
```

Comment:
This chart repo also needs to be added to the `config/repo-values.yaml` as well. Can you please add it there?

Reply:
Done 👍